packages/manager/apps/web/client/app/domain-operation/domain-operation.html

```diff
@@ -1,262 +1,23 @@
-<div ui-view>
-    <oui-header
-        heading="{{:: 'domains_operations_dashboard_title' | translate }}"
-    >
-    </oui-header>
-
-    <div data-ovh-alert="{{alerts.main}}"></div>
-
-    <p data-translate="domains_operations_dashboard_info"></p>
-
-    <div class="mb-3">
-        <form
-            class="form-inline d-md-flex justify-content-md-end"
-            name="domainsOperationsForm"
+<oui-header heading="{{:: 'domain_operations_dashboard_title' | translate }}">
+</oui-header>
+
+<div class="oui-header">
+    <oui-header-tabs>
+        <oui-header-tabs-item
+            href="{{ $ctrl.domainOperationLink }}"
+            active="!$ctrl.isDns()"
```

Comment:
i think it's more efficient to base on state sref because if one day we have to add new tab it's more easier.
For example:
```suggestion
            active="$ctrl.getTabItemUrl('dns') === $ctrl.currentActiveLink()"
```

Reply:
done